src/status_im/contexts/wallet/create_account/new_keypair/keypair_name/view.cljs

```diff
@@ -42,4 +50,20 @@
             :label           (i18n/label :t/keypair-name)
             :char-limit      keypair-name-max-length
             :auto-focus      true
-            :on-change-text  #(reset! keypair-name %)}]]]))))
+            :on-change-text  (fn [value]
+                               (reset! keypair-name value)
+                               (cond
+                                 (> (count value) keypair-name-max-length) (set-error? 1)
+                                 (has-emojis? value)                       (set-error? 2)
+                                 (has-special-characters? value)           (set-error? 3)
+                                 :else                                     (set-error? nil)))
+            :error?          error?}]
+          (when error?
+            [rn/view
+             {:style style/error-container}
+             [quo/icon :i/info {:color (colors/theme-colors colors/danger-50 colors/danger-60 theme)}]
```

Comment:
same for danger colors here and below. Can you use color/resolve-color method instead?